Source/AliveLibAO/Slig.cpp

```diff
@@ -3830,8 +3886,59 @@ __int16 Slig::Brain_Death_46C3A0()
 
 __int16 Slig::Brain_DeathDropDeath_46C5A0()
 {
-    NOT_IMPLEMENTED();
-    return 0;
+    if (field_10E_brain_state == 0)
+    {
+        Sfx_Slig_GameSpeak_46F560(10u, 0, field_110_pitch_min, this);
+        field_114_timer = gnFrameCount_507670 + 60;
+        return 1;
+    }
+
+    if (field_10E_brain_state - 1)
```

Comment:
if (field_10E_brain_state == 0)
if (field_10E_brain_state - 1)
if ((field_10E_brain_state - 1) == 1 

is a switch case on field_10E_brain_state that needs de-tangling